### PR TITLE
Add ArcticDEM and REMA yamls

### PR DESCRIPTION
**Pull request by clairecporter**
*Issue #, if available:*

*Description of changes:*
Add yaml files from PGC for the ArcticDEM and REMA projects.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: datasets/pgc-rema.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Name: REMA
+Name: REMA DEM
```
Comment (jeffejefe) [inline datasets/pgc-rema.yaml:1]: @clairecporter The name needs to be at least 5 chars in length so added "DEM".

Comment (clairecporter) [reply datasets/pgc-rema.yaml]: I used the full dataset name instead.  Thanks for figuring that out.

**Review by jeffejefe — CHANGES_REQUESTED** [review]

File: datasets/pgc-arcticdem.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+Name: ArcticDEM
+Description:  ArcticDEM - 2m GSD Digital Elevation Models (DEMs) and mosaics from 2007 to the present
```
[the remark this answers is not in the document]
Comment (clairecporter) [reply datasets/pgc-arcticdem.yaml]: Done. You are welcome to review again.

File: datasets/pgc-rema.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+Description: The Reference Elevation Model of Antarctica - 2m GSD Digital Elevation Models (DEMs) and mosaics from 2009 to the present
```
[the remark this answers is not in the document]
Comment (clairecporter) [reply datasets/pgc-rema.yaml]: Done. You are welcome to review again.

**Pull request merged**